scripts/cicd/verify-gather-logs-operations.py

```diff
@@ -73,9 +89,15 @@ def gather_logs(command):
                         required=True, type=valid_krbid)
     parser.add_argument('-c', dest='cluster', help="Cluster name",
                         required=True, choices=VALID_CLUSTER_NAMES)
+    parser.add_argument('-n', dest='nodes', help="Nodes to gather logs from",
```

Comment:
From a security perspective, I'm happy with either.  I feel that "-n node1 node2" will be open to misunderstanding later if "node2" in that example could be considered an appending on the -n list, or an arg in it's own right.